ee/clickhouse/migrations/0005_kafka.py

```diff
@@ -1,13 +1,28 @@
 from infi.clickhouse_orm import migrations
 
-from ee.clickhouse.sql.elements import KAFKA_ELEMENTS_GROUP_TABLE_SQL, KAFKA_ELEMENTS_TABLE_SQL
-from ee.clickhouse.sql.events import KAFKA_EVENTS_TABLE_SQL
-from ee.clickhouse.sql.person import KAFKA_PERSONS_DISTINCT_ID_TABLE_SQL, KAFKA_PERSONS_TABLE_SQL
+from ee.clickhouse.sql.elements import (
+    ELEMENTS_GROUP_TABLE_MV_SQL,
+    ELEMENTS_TABLE_MV_SQL,
+    KAFKA_ELEMENTS_GROUP_TABLE_SQL,
+    KAFKA_ELEMENTS_TABLE_SQL,
+)
+from ee.clickhouse.sql.events import EVENTS_TABLE_MV_SQL, KAFKA_EVENTS_TABLE_SQL
+from ee.clickhouse.sql.person import (
+    KAFKA_PERSONS_DISTINCT_ID_TABLE_SQL,
+    KAFKA_PERSONS_TABLE_SQL,
+    PERSONS_DISTINCT_ID_TABLE_MV_SQL,
+    PERSONS_TABLE_MV_SQL,
+)
 
 operations = [
     migrations.RunSQL(KAFKA_EVENTS_TABLE_SQL),
     migrations.RunSQL(KAFKA_ELEMENTS_TABLE_SQL),
     migrations.RunSQL(KAFKA_ELEMENTS_GROUP_TABLE_SQL),
     migrations.RunSQL(KAFKA_PERSONS_TABLE_SQL),
     migrations.RunSQL(KAFKA_PERSONS_DISTINCT_ID_TABLE_SQL),
+    migrations.RunSQL(EVENTS_TABLE_MV_SQL),
```

Comment:
Did you mean to put them in the same migration? Are you going to force run this migration again?

Reply:
I haven't run the migration on prod yet. Only manually creating tables to test. This all has only been tested locally.

Reply:
got it. i thought the migrations that exist currently were all run in prod already